tests/wrappers/utils.py

```diff
@@ -0,0 +1,11 @@
+from typing import Union
+
+import gym
+
+
+def has_wrapper(wrapped_env: Union[gym.Wrapper, gym.Env], wrapper_type: type):
```

Comment:
The Union is redundant, `gym.Wrapper` subclasses `gym.Env`

Reply:
And a return type hint would be nice just out of principle

Reply:
Fixed in 02c7957d4b2851c99fb06c7afbfee055eccdf09c